kroxylicious-kms-provider-aws-kms-test-support/src/main/java/io/kroxylicious/kms/provider/aws/kms/AwsKmsTestKmsFacade.java

```diff
@@ -139,7 +140,7 @@ public TestKekManager getTestKekManager() {
         return new AwsKmsTestKekManager();
     }
 
-    class AwsKmsTestKekManager implements TestKekManager {
+    protected class AwsKmsTestKekManager implements TestKekManager {
```

Comment:
why protected?

Reply:
Do you think the `Abstract` class is still needed? All the `Kube` test kek managers has been deleted and not needed anymore.

Reply:
VaultTestKmsFacade knows about Docker.   KubeVaultTestKmsFacade extends VaultTestKmsFacade.  That doesn't make sense, as logically KubeVaultTestKmsFacade has nothing to do with Docker.

So I'd push the common parts up into the abstract (the TestKekManager etc) and change the hierarchy so that VaultTestKmsFacade and KubeVaultTestKmsFacade are siblings.

AbstractVaultTestKmsFacade <- VaultTestKmsFacade
AbstractVaultTestKmsFacade <- KubeVaultTestKmsFacade